extensions/wikia/ArticleVideo/ArticleVideoController.class.php

```diff
@@ -29,4 +29,16 @@ public function related() {
 			$this->skipRendering();
 		}
 	}
+
+	public function labels() {
+		$videoId = $this->getVal( 'videoId', null );
+
+		if ( empty( $videoId ) ) {
+			throw new MissingParameterApiException( 'videoId' );
+		}
+
+		$api = OoyalaBacklotApiService::getInstance();
+		$this->response->setFormat( WikiaResponse::FORMAT_JSON );
+		$this->response->setVal( 'labels', $api->getLabels( $videoId ) );
```

Comment:
It's a request to 3rd party API. Can we handle errors and make sure site won't be down when there are any problems on Ooyala side?